src/components/CardEditor/_card-editor.scss

```diff
@@ -31,3 +31,8 @@
     }
   }
 }
+
+.#{$iot-prefix}--card-gallery-list__icon > svg {
+  height: 34px;
+  width: 40px;
+}
```

Comment:
Will this contort icons that don't match the aspect ratio of the `@carbon/icons-pictogram` svgs?

Reply:
![image](https://user-images.githubusercontent.com/1293444/102498932-3408ab00-4040-11eb-8913-066ca044735b.png)

Tested out a few icons; Code16 and Basketball32 to test if it squashes or looks alright. Otherwise I could have it up to the developer to add their own style to the icons. Which way is more preferable?

Reply:
Looks fine to me - the style is always overridable if the consumer needs to. 👍